prov/sockets/src/sock_progress.c

```diff
@@ -2355,7 +2356,8 @@ static int sock_pe_progress_rx_ep(struct sock_pe *pe, struct sock_ep_attr *ep_at
 			SOCK_EPOLL_WAIT_EVENTS, 0);
 	if (num_fds < 0 || num_fds == 0) {
 		if (num_fds < 0)
-			SOCK_LOG_ERROR("poll failed: %s\n", strerror(errno));
+			SOCK_LOG_ERROR("poll failed: %s\n",
+				       strerror(ofi_sockerr()));
```

Comment:
mixing epoll with sockerr

Reply:
`fi_epoll_wait` is mapped to:
- `epoll` if `epoll` is available
- `poll` if `poll` is unavailable